
class AuthException(Exception):
    pass

class OperationalException(Exception):
    pass